deploy/helm/sumologic/values.yaml

```diff
@@ -210,13 +203,21 @@ fluent-bit:
     systemd:
       enabled: true
 
+  parsers:
+    enabled: true
+    # This regex matches the first line of a multiline log starting with a date of the format : "2019-11-17 07:14:12"
+    regex:
+      - name: multi_line
+        regex: (?<log>^{"log":"\d{4}-\d{1,2}-\d{1,2} \d{2}:\d{2}:\d{2}.*)
+      
```

Comment:
@vsinghal13 are we using these values somewhere else? because we have defined same thing in `deploy/helm/fluent-bit-overrides.yaml`

Reply:
The `fluent-bit-overrides.yaml` is an auto-generated file for all the fluent-bit values that have been overridden in `values.yaml` . This is done for non-helm users.